prow/clonerefs/run.go

```diff
@@ -73,7 +73,7 @@ func (o Options) Run() error {
 	}
 
 	for _, ref := range o.GitRefs {
-		input <- ref
+		input <- &ref
```

Comment:
This is incorrect, this sends the same pointer to input multiple times. ref #10093

Reply:
Agree with Ben, nice catch... can we add a unit test that would have detected this problem?